c/validation/adbc_validation.cc

```diff
@@ -945,6 +945,65 @@ void ConnectionTest::TestMetadataGetObjectsConstraints() {
   // TODO: can't be done portably (need to create tables with primary keys and such)
 }
 
+void ConstraintTest(AdbcGetObjectsConstraint* constraint, std::string key_type,
```

Comment:
Can we use `const` here?

`const AdbcGetObjectsConstraint* constraint, const std::string& key_type, const std::vector<std::string>& columns`